webapp/src/App.svelte

```diff
@@ -2,83 +2,230 @@
-      on:click={() => (timeseries = "MONTHLY")}
-    >
-      Monthly
-    </button>
-  </div>
-  {#key timeseries}
-    {#await fetchData()}
-      <div class="flex flex-col items-center text-white">
-        <p>Loading...</p>
-      </div>
-    {:then chart}
-      <div class="flex flex-col items-center">
-        <Ferdous width={420} height={420} {graphData} />
+
+    <div class="flex justify-center">
+      <div class="" on:focusout={handleDropdownFocusLoss}>
+        <button class="btn m-1" on:click={handleDropdownClick}>
+          {#if isDropdownOpen}
+            <svg
+              xmlns="http://www.w3.org/2000/svg"
+              fill="none"
+              viewBox="0 0 24 24"
+              class="inline-block h-8 w-8 stroke-current bg-slate-500 rounded-lg"
+            >
+              <title>Close Dropdown</title>
+              <path
+                stroke-linecap="round"
+                stroke-linejoin="round"
+                stroke-width="2"
+                d="M6 18L18 6M6 6l12 12"
+              />
+            </svg>
+          {:else}
+            <div class="h-10 w-min bg-slate-500 rounded-xl p-2 font-bold text-white">
+              <p>{timeseries}🔽</p>
+            </div>
+          {/if}
+        </button>
+        <ul
+          class="bg-slate-500 rounded-box w-min rounded-lg py-2 font-bold"
+          style:visibility={isDropdownOpen ? "visible" : "hidden"}
+        >
+          <li class="hover:bg-slate-800 text-left ">
+            <button
+              class="btn text-white p-1 w-full text-left"
+              on:click={() => (timeseries = "DAILY")}
+              on:click={handleDropdownClick}>DAILY</button
+            >
+          </li>
+          <li class="hover:bg-slate-800">
+            <button
+              class="btn text-white p-1 w-full text-left"
+              on:click={() => (timeseries = "WEEKLY")}
+              on:click={handleDropdownClick}>WEEKLY</button
+            >
+          </li>
+          <li class="hover:bg-slate-800">
+            <button
+              class="btn text-white p-1 w-full text-left"
+              on:click={() => (timeseries = "MONTHLY")}
+              on:click={handleDropdownClick}>MONTHLY</button
+            >
+          </li>
+        </ul>
       </div>
-    {/await}
-  {/key}
+    </div>
+  </div>
+  <div class="">
+    {#key timeseries}
+      {#await fetchData()}
+        <div class="flex flex-col items-center text-white">
+          <p>Loading...</p>
+        </div>
+      {:then chart}
+        <div class="sm:grid sm:grid-rows-3 sm:grid-flow-col sm:gap-3">
+          <div class="flex flex-col items-center">
+            <p class="text-white">QQQ</p>
+            <Ferdous
+              width={420}
+              height={420}
+              theme={THEME}
+              graphData={graphData.filter((item) => item.symbol == "QQQ")}
+            />
+          </div>
+
+          <div class="flex flex-col items-center">
+            <p class="text-white">PANW</p>
```

Comment:
This <p> tag containing the symbol and the div class wrapped around it and the `Ferdous` component should lal be within the `Ferdous` component